android/java/org/chromium/chrome/browser/ntp/BraveNewTabPage.java

```diff
@@ -74,19 +74,15 @@ public BraveNewTabPage(Activity activity,
             for (TabModel tabModel : tabModelSelector.getModels()) {
                 if (tabModel.getProfile() != null) {
                     TemplateUrlService templateUrlService =
-                            BraveTemplateUrlServiceFactory.getForProfile(tabModel.getProfile());
-                    if (templateUrlService != null) {
```

Comment:
I wonder why this check is no longer necessary.

Reply:
The upstream function doesn't return null so it shouldn't be necessary to check it anymore.